libraries/EEPROM/src/EEPROM.h

```diff
@@ -69,11 +71,11 @@ Error : EEPROM start address must be divisble by 8192
         //to 2048 if needed
```

Comment:
These particular comments still make it sounds like you increase the size via changing this constant and that it is limited to 2048. This line should be updated to reflect the existence of the setLength() member function, i.e. "// up to 8192 (AP3_EEPROM_MAX_LENGTH)  if needed, via the EEPROM.setLength()". (You could just cut and paste or refer to the very clear comments from Example2_AllFunctions.ino)